mmdet3d/datasets/semantickitti_dataset.py

```diff
@@ -60,6 +67,32 @@ def __init__(self,
             filter_empty_gt=filter_empty_gt,
             test_mode=test_mode)
 
+        self.ignore = np.array(ignore, dtype=np.int64)
+        self.n_classes = len(self.CLASSES)
+
+        self.include = np.array(
+            [n for n in range(self.n_classes) if n not in self.ignore],
+            dtype=np.int64)
+        self.offset = offset
+        self.eps = 1e-15
+        self.min_points = min_points
+        self.reset()
+        self.global_cfg = mmcv.load(
+            osp.join(self.data_root, 'semantic-kitti.yaml'))
```

Comment:
where is the yaml file?

Reply:
The related parameters "learning_map" in the yaml file have been added as the global setting in `class SemanticKITTIDataset`